shared/files/server/teamcity-backup

```diff
@@ -7,11 +7,15 @@ TEAMCITY_DIR={{TEAMCITY_DIR}}
 TEAMCITY_DATA_DIR={{TEAMCITY_DATA_DIR}}
 TEAMCITY_USER={{TEAMCITY_USER}}
 TEAMCITY_GROUP={{TEAMCITY_GROUP}}
+TEAMCITY_VAGRANT_DIR={{TEAMCITY_VAGRANT_DIR}}
 
 # I don't know why JAVA HOME is not coming through appropriately but it's
 # something to think about
 export JAVA_HOME=/usr/lib/jvm/java-8-oracle
-$TEAMCITY_DIR/bin/maintainDB.sh backup -C -D -L -P -U -A $TEAMCITY_DATA_DIR
-chown -R $TEAMCITY_USER:$TEAMCITY_GROUP $TEAMCITY_DATA_DIR/backup
-chmod 755 $TEAMCITY_DATA_DIR/backup
-chmod 444 $TEAMCITY_DATA_DIR/backup/*.zip
+
```

Comment:
`--all` is a more human readable alias for `-C -D -L -P -U` and `--data-dir` is a more human readable alias for `-A`